test/shared/setup.ts

```diff
@@ -573,3 +573,64 @@ export async function starkTxAuthSetup() {
     vanillaExecutionStrategy,
   };
 }
+
+export async function ethereumSigSetup() {
+  const account = await starknet.deployAccount('OpenZeppelin');
+
+  const vanillaSpaceFactory = await starknet.getContractFactory('./contracts/starknet/Space.cairo');
+  const vanillaVotingStategyFactory = await starknet.getContractFactory(
+    './contracts/starknet/VotingStrategies/Vanilla.cairo'
+  );
+  const starknetSigAuthFactory = await starknet.getContractFactory(
+    './contracts/starknet/Authenticators/EthSig.cairo'
+  );
+  const zodiacRelayerFactory = await starknet.getContractFactory(
+    './contracts/starknet/ExecutionStrategies/ZodiacRelayer.cairo'
```

Comment:
I was generally using vanilla contracts for the modules that were not part of the current test, just to simplify things.

Reply:
definitely right